onnxruntime/core/graph/model.h

```diff
@@ -96,6 +96,8 @@ class Model {
   // Get model's serialization proto data.
   ONNX_NAMESPACE::ModelProto ToProto();
 
+  bool InputsUseFP16();
```

Comment:
>InputsUseFP16 [](start = 7, length = 13)

Can we use the same name as the graph.h api: DoInputsHaveFP16

Reply:
This matches the public method InputsUseFP16() in Graph.h api

---
In reply to: [382313343](https://github.com/microsoft/onnxruntime/pull/3061#discussion_r382313343) [](ancestors = 382313343)